app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -218,10 +218,82 @@
 }
 
 .boundary-line {
+  border-bottom: 2px solid #aaa;
   padding-top: 30px;
+}
+
+.panel-boundary-bottom {
   border-bottom: 2px solid #aaa;
+  margin-left: 5px;
+  margin-right: 5px;
+  padding-top: 5px;
 }
 
 .book-title-table {
   max-width: 600px;
 }
+
+.panel-title {
+  font-size: 23px;
+  margin-left: 10px;
+  margin-top: 5px;
+  padding-bottom: 5px;
+}
+
+.book-search-panel-left {
+  border-right: 1px solid #ddd;
+  float: left;
+  margin-bottom: 5px;
+  margin-top: 5px;
+  padding-right: 10px;
+  width: 49%;
+}
+
+.book-search-panel-right {
+  border-left: 1px solid #ddd;
+  float: left;
+  margin-bottom: 5px;
+  margin-left: -1px;
+  margin-top: 5px;
+  padding-right: 10px;
+  width: 49%;
+}
+
+.book-search-fields {
+  margin-left: 0px;
+  margin-right: 0px;
+  margin-top: 10px;
+  width: 100%;
+}
+
+.book-search-rows {
+  margin: 0px;
+}
+
+.book-search-list {
+  margin: 5px;
+  list-style-type: none;
+}
+
+.book-search-li-icon {
+  color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.